src/simplewallet/simplewallet.cpp

```diff
@@ -5369,7 +5369,7 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
   const uint64_t last_block_height = m_wallet->get_blockchain_current_height();
 
   std::list<std::pair<crypto::hash, tools::wallet2::payment_details>> payments;
-  m_wallet->get_payments(payments, 0, m_current_subaddress_account);
+  m_wallet->get_payments(payments, 0, (uint64_t)-1, m_current_subaddress_account);
```

Comment:
Does uint64_t initialise to the same value on every platform?

Reply:
If GCC conforms to the C11 Standard, then yes: https://stackoverflow.com/a/18303778

Also, this expression `(uint64_t)-1` is commonly used in the codebase.